user-guide/Advanced_Functionality/Databases/Database_configuration_in_Cube/Configuring_the_database_settings_in_Cube.md

```diff
@@ -81,7 +83,41 @@ In case a separate Cassandra cluster is used per DMA, configure the settings as
 
 1. Click *Save*.
 
-## [Legacy MySQL or SQL database](#tab/tabid-3)
+***
+
+## Amazon Keyspaces database
```

Comment:
For the user, it may be handier to have these under separate titles, instead of having one combined procedure for both where they need to sift through the information to pick what is relevant for them.

Also, for a Cassandra cluster database, right now we say the prefix should be specified with the **Name** setting, while for AWS we say it's the **Keyspace prefix**. Could you just confirm that this setting is indeed different in Cube?

Reply:
Okay Fair enough.

And yes with the proposed cube changes, the **Name** setting was changed to **Keyspace prefix** for Amazon Keyspaces **and** Cassandra cluster to better portray their true meaning. However for compatibilty reasons, they are unchanged in db.xml under the <DB> tag